frontend/src/containers/pages/InvalidRolePage.jsx

```diff
@@ -0,0 +1,34 @@
+import BackButton from "../../components/BackButton";
+
+function InvalidRolePage() {
+  const currentUserRole = "Doctor";
+  const rolesWithAccess = ["Nurse", "Patient"];
+
+  const containerStyle = {
+    fontFamily: "Sitka Text, sans-serif",
+    fontWeight: "600",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+    minHeight: "100vh",
+    textAlign: "center",
+  };
+
+  const textMargin = {
+    margin: "40px 0",
+  };
+
+  return (
+    <div style={containerStyle}>
+      <h1 style={textMargin}>
+        Someone else is playing through this section of the scenario!
+      </h1>
+      <p style={textMargin}>Please wait for your role: {currentUserRole}</p>
+      <BackButton />
+      <p style={textMargin}>Roles with access: {rolesWithAccess.join(", ")}</p>
```

Comment:
Hmm, in the wireframe, this text is near the bottom. As you seem to have done this with flex boxes (nice!), think about how you can use multiple divs to group and position in a 'responsive' way.

![image](https://github.com/UoaWDCC/VPS/assets/100455176/4329bcfe-e20c-4a6d-b936-5d584b023469)

Reply:
![image](https://github.com/UoaWDCC/VPS/assets/136278107/b43a7cfb-89b3-4bec-a65c-78485eb27770)

updated, looks like that now.